modules/imaging_uploader/php/NDB_Menu_Filter_imaging_uploader.class.inc

```diff
@@ -170,6 +171,10 @@ class NDB_Menu_Filter_Imaging_Uploader extends NDB_Menu_Filter_Form
             );
 
             if ($ImagingUploaderAutoLaunch) {
+                // Instanciate the server process module to autoload
+                // its namespace classes
+                $spm_module = \Module::factory('server_processes_manager');
```

Comment:
This variable is unused and phpmd will complain.. you can just update the comment to explain that you're relying on a side-effect of the call and not assign it to anything.